src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php

```diff
@@ -69,13 +69,42 @@ public function compileDropDatabaseIfExists($name)
     /**
      * Compile the query to determine if a table exists.
      *
+     * @deprecated Will be removed in a future Laravel version.
+     *
      * @return string
      */
     public function compileTableExists()
     {
         return "select * from sys.sysobjects where id = object_id(?) and xtype in ('U', 'V')";
     }
 
+    /**
+     * Compile the query to determine the tables.
+     *
+     * @return string
+     */
+    public function compileTables()
+    {
+        return 'select t.name as name, SCHEMA_NAME(t.schema_id) as [schema], sum(u.total_pages) * 8 * 1024 as size '
```

Comment:
@hafezdivandari Why are you multiplying by 8 specifically here?

Reply:
Because each page is 8 bytes according to [docs](https://learn.microsoft.com/en-us/sql/relational-databases/pages-and-extents-architecture-guide?view=sql-server-ver16#:~:text=SQL%20Server%20writes%20all%20data%20rows%20on%20pages%2C%20and%20all%20data%20pages%20are%20the%20same%20size%3A%208%20KB.).

Reply:
@hafezdivandari But why not just multiply by `8192` directly instead then?

Reply:
because that's more readable and easier to understand imo.